sdk/src/Core/Amazon.Util/Internal/_netstandard/InternalSDKUtils.netstandard.cs

```diff
@@ -25,14 +25,30 @@
 using System.Text;
 
 namespace Amazon.Util.Internal
-{
+{        
+    /// <summary>
+    /// An interface for <see cref="RuntimeInformationWrapper"/> which allows for mocking the <see cref="RuntimeInformation"/> class.
+    /// A wrapper is necessary because the class is static.
+    /// </summary>
+    internal interface IRuntimeInformationWrapper
```

Comment:
Does this interface need to marked `internal`? It's already in the `Internal` namespace, and in my opinion made this PR much more complicated than necessary.

Reply:
Norm asked me to make these internal and utilize `InternalsVisibleTo`😅.

Reply:
I'm fine with either approach. My intention was I wanted the new interface to be treated as internal. Probably not a bad thing to get our unit tests project setup to access internals for future testing.

Reply:
I agree it's not a bad thing to have it set up so that we can use it in the future.